seq2science/rules/configuration_generic.smk

```diff
@@ -300,7 +311,7 @@ with FileLock(layout_cachefile_lock):
         layout_cache = {}
 
 
-    trace_tp = ThreadPool(20)
+    trace_tp = ThreadPool(40)
```

Comment:
should this be `config["threads"]`?

Reply:
Well Python's threads do not really work well in the sense that they won't be executed in "parallel". 40 threads together will never use more than 1 cpu thread. However when threads are waiting for stuff (e.g. waiting for websites to reply), non-waiting threads can continue with stuff. So the threads here are in a sense not the same as the threads of Snakemake. 

40 might be a bit of overkill though, 20 should be fine I think, I'll revert that back.